server/main.go

```diff
@@ -29,5 +72,7 @@ func main() {
 	// Routes
 
 	// Start server
+	e.POST("/post", addPost)
+	e.GET("/post", showPost)
```

Comment:
API route は、複数の投稿を取得したいので `/posts` がいいと思います！（`addPost` はひとつしか追加しないので単数形で大丈夫です）
```suggestion
	e.POST("/posts", addPost)
	e.GET("/posts", showPosts)
```

Reply:
あと大したことないことなんですが、このへんは `// Routes` コメントの直後に書いてくれるとありがたいです！